tests/integration/profiling/rule_based_profiler/test_profiler_user_workflows.py

```diff
@@ -463,13 +471,21 @@ def test_alice_expect_column_values_to_not_match_regex_auto_yes_default_profiler
     assert not result.success
 
     expectation_config_kwargs: dict = result.expectation_config.kwargs
+
+    assert expectation_config_kwargs["regex"] in [
```

Comment:
Curious about this assertion: Why are we checking for more patterns than before?

Reply:
@Shinnnyshinshin It turns out that we have several patterns with the maximum winning score of 1.0, so any one of them are applicable.  The system is unable to unambiguously choose one winner, so depending on random factors, sometimes a different one of these included in the list shows up as the winner.  Thanks.

Reply:
sounds good